consensus/replay_test.go

```diff
@@ -893,11 +906,16 @@ func TestHandshakePanicsIfAppReturnsWrongAppHash(t *testing.T) {
 		proxyApp := proxy.NewAppConns(clientCreator)
 		err := proxyApp.Start()
 		require.NoError(t, err)
-		defer proxyApp.Stop()
+		t.Cleanup(func() {
+			if err := proxyApp.Stop(); err != nil {
+				t.Error(err)
+			}
+		})
 
 		assert.Panics(t, func() {
 			h := NewHandshaker(stateDB, state, store, genDoc)
-			h.Handshake(proxyApp)
+			err = h.Handshake(proxyApp)
```

Comment:
we can use `_ = h.Handshake` because it's assumed to panic

Reply:
We still want to be notified of errors, it makes it easies to understand test failures.